processes/general/deep_link/README.md

```diff
@@ -0,0 +1,66 @@
+# OPES Deep Link / QR Code Specification
+
+In order to generate activity between an Access Node (AN) and a Personal Node (PN),
+we use a deep link configured and provided by the AN. With that link, the PN can
+initiate communication with the AN and, between the two, generate and send an
+activity tally into the system.
+```
+IMPORTANT: This document describes the current specification,
+           but you should expect that this specification will change in the future.
+```
+## Deep Link Query Parameters
+
+In order to generate the deep link and QR code, the following values must be
+determined.
+
+| Value | Description |
+| ---- | ---- | 
+| host [REQUIRED] | The deep link host. Currently, the only valid value is `mobile.opes.pe`. |
+| action [REQUIRED] | The action to initiate between AN and PN. Currently, the only valid value is `check-in`. |
```

Comment:
@eahrold are other values such as `sign-in` supported?

Reply:
the mobile app will handle url's to "sign-in", but I'm not sure any of the other SSO feature made it to prod.

So this is probably fine for now.